docs/changelog/89106.yaml

```diff
@@ -4,3 +4,15 @@ area: SQL
 type: enhancement
 issues:
  - 88764
+highlight:
+  title: Speed up SQL queries by not tracking total hits by default
+  body: |-
+    SQL query translator now explicitly sets track_total_hits to false when
+    not needed.
+    This has a significant impact on SQL query performance in cases where total hits
+    are not needed to calculate the final result, in particular when the cost of evaluation
+    of a single document is particularly high (eg. queries that involve script evaluation)
+    and in queries with a small LIMIT value.
+    In our tests, on some specific queries, we see a speed-up of more than 50%,
+    with peaks of ~95% (from 600ms to 20ms).
```

Comment:
I think specific timing values tend to not age well (and prompt questions about the setup), but if we do this generally, it's all good (an alternative being just "dramatic performance improvements" etc.).

Reply:
I like the wording, which says "our tests" and "specific queries" to set expectations correctly, so that users do not assume that it would necesseraly mean the same for their queries.